WolvenKit/Views/Editors/AutomaticNodifyEditor.cs

```diff
@@ -30,14 +30,17 @@ protected override void OnMouseMove(MouseEventArgs e)
 
             if (CurrentMousePosition != PreviousMousePosition)
             {
-                MouseLocation = TransformPosition(CurrentMousePosition);
 
+                //MouseLocation = TransformPosition(CurrentMousePosition);
+                // ^ `TransformPosition` was removed from NodifyEditor. Rather than trying to figure out the "correct" way to fix it,
+                // I hard coded it to what the definition of the function _was_ just to get things to build correctly on Nodify 3.0.0
+                MouseLocation = new Point((ViewportLocation.X + CurrentMousePosition.X) / ViewportZoom, (ViewportLocation.Y + CurrentMousePosition.Y) / ViewportZoom);
```

Comment:
All coordinates that the editor works with should be inside the `ItemsHost`. Translating coordinates is not needed if, for example, you want to move the viewport to a certain location inside the editor's space. (like focusing a node)

`MouseLocation =  e.GetPosition(ItemsHost);` should do the trick.

Reply:
:naruhodo: I will add that in. Thanks!